docs/DeveloperGuide.md

```diff
@@ -604,6 +636,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case resumes at step 2.
 
+**Use case: Add a tag to a task from task list**
+
+**MSS**
+
+1. User requests to list tasks
+2. Coach2K22 shows a list of tasks
+3. User requests to add a tag to a task of a specified index in the task list
+4. Coach2k22 shows the updated details of the task list
+
+  Use case ends.
+
+**Extensions**
+
+* 3a. The task list is empty.
+
+  Use case ends.
+
+**Use case: Remove a tag from a task in task list**
+
+**MSS**
+
+1. User requests to list tasks
+2. Coach2K22 shows a list of tasks
+3. User requests to remove a tag from a task of a specified index in the task list
+4. Coach2K22 shows the updated details of the task list
+
+   Use case ends.
+
+**Extensions**
+
```

Comment:
Might want to include more extensions for this e.g. invalid index, empty list.

Reply:
Added more extensions